fairing/backend/kubernetes/manager.py

```diff
@@ -113,7 +114,7 @@ def log(self, name, namespace, extra_label_selector=None):
             logger.error("error getting status for {} {}".format(name, str(e)))
         if tail:
             try:
-                for chunk in tail.stream(MAX_STREAM_BYTES):                    
-                    print(chunk.rstrip())
+                for chunk in tail.stream(MAX_STREAM_BYTES):
+                    print(chunk.rstrip().decode('utf8'))
```

Comment:
We were outputting byte arrays in python3 notebooks, this makes the print statement pretty.